### DISCUSSION TITLE
Please add "Center Two Thirds" and "Center Three Fourths"

### DISCUSSION THREAD

**Discussion by debajit** [rxhanson/Rectangle#549] [Ideas]
Please add "Center Two Thirds" and "Center Three Fourths" as options for window sizes.

**Comment by rxhanson** [comment]
Center 2/3 is a repeated execution of the half command (this is the default behavior in the current version). I have plans to make some changes in an upcoming release to allow developers to add more size/position commands. You can also dig through the readme for terminal commands for custom centered sizes, or you could use the custom shortcut builder in my paid superset of Rectangle, Hookshot (https://hookshot.app)

**Reply by debajit** [reply]
> Center 2/3 is a repeated execution of the half command (this is the default behavior in the current version).

Could you explain the steps I would need to take to achieve Center 2/3 by repeating the half command? 

(I tried running "Left Half" (as an example) twice, but that only ended up moving my halved window to my other monitor, and did not resize it to Center 2/3)

**Reply by N-Molham** [reply]
@debajit I think Ryan is referring to this https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#adjust-behavior-on-repeated-commands ... I just opened the terminal and ran this command `defaults write com.knollsoft.Rectangle centerHalfCycles -int 1` then close `cmd+Q` the app then relaunch it and now hitting the same keyboard shortcuts for center half cycles throw half size then two third then one third.
